src/main/resources/application.yaml

```diff
@@ -79,4 +79,6 @@ scheduler:
 
 userprofile:
   api:
-    url: ${USER_PROFILE_URL:http://rd-user-profile-api-demo.service.core-compute-demo.internal}
\ No newline at end of file
+    url: ${USER_PROFILE_URL:http://rd-user-profile-api-demo.service.core-compute-demo.internal}
+
+loggingComponentName: " RD_Profile_Sync "
```

Comment:
no need to add quotes i think ..yaml automatically picks as a string..can you please verify once

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-profile-sync/pull/277/commits/7a9d100a178f09754ac0b383dc30f585d9bc3e38